tests/integration/FirehoseIntegrationTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * This file is part of graze/queue.
+ *
+ * Copyright (c) 2015 Nature Delivered Ltd. <https://www.graze.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license https://github.com/graze/queue/blob/master/LICENSE MIT
+ *
+ * @link    https://github.com/graze/queue
+ */
+
+namespace Graze\Queue;
+
+use Aws\ResultInterface;
+use Aws\Firehose\FirehoseClient;
+use Graze\Queue\Adapter\FirehoseAdapter;
+use Mockery as m;
+use Mockery\MockInterface;
+use PHPUnit_Framework_TestCase as TestCase;
+
+class FirehoseIntegrationTest extends TestCase
+{
+    /** @var string */
+    private $queueName;
```

Comment:
The property `$queueName` is not used and could be removed.

Reply:
done